### PR TITLE
Implemented removing tokens in `DeleteTokenSlice`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #57 
Added `AuthTokens` to `DeleteTokenSlice`, removed old tokens map, corrected test for `DeleteTokenSlice` and enabled IT.

### CODE REVIEW

File: src/main/java/com/artipie/conda/http/DeleteTokenSlice.java [hunk 1]
```diff
@@ -48,24 +47,30 @@ public Response response(final String line,
...
-                            status = RsStatus.CREATED;
-                        }
-                        res = new RsWithStatus(status);
+                () -> new RqHeaders(headers, Authorization.NAME)
+                    .stream().findFirst().map(Authorization::new)
+                    .map(auth -> new Authorization.Token(auth.credentials()).token())
+            ).thenCompose(
+                tkn -> {
+                    final CompletionStage<Response> res;
+                    if (tkn.isPresent()) {
```
Comment (g4s8) [inline src/main/java/com/artipie/conda/http/DeleteTokenSlice.java:56]: You can simplify this to `tkn.map(token -> this.tokens.remove(token).thenApply(...)).orElse(new RsWithHeaders(UNAUTHORIZED))`

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova just one comment

**Comment by olenagerasimova** [comment]
@g4s8 thanks, fixed

**Comment by olenagerasimova** [comment]
@g4s8 please, check

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**